atlassian/jira/tool.gpt

```diff
@@ -191,4 +191,8 @@ Jira
 
 ---
 !metadata:*:oauth
-atlassian
\ No newline at end of file
+atlassian
+
+---
+!metadata:*:supportsOAuthTokenPrompt
```

Comment:
The description of this PR references an `oauthPATSupported` field. The field here is named differently. Is this intentional?

Reply:
I think that's changed in this PR https://github.com/obot-platform/obot/pull/1387